examples/pplm/run_pplm.py

```diff
@@ -508,6 +518,13 @@ def generate_text_pplm(
 
         pert_logits, past, pert_all_hidden = model(last, past=pert_past)
         pert_logits = pert_logits[:, -1, :] / temperature  # + SMALL_CONST
+
+        for j in set(output_so_far[0].tolist()):
```

Comment:
would rename `j` to `token_idx` for readability

Reply:
Fixed